### PR TITLE
Transfer software

### PR DESCRIPTION
**Pull request by jmvera255**
revision of Transfer Software With HTCondor, no longer uses a tutorial. I plan to revise the Transfer Data With HTCondor to follow the same outline but only once this pull request has been reviewed and approved.

Would also like to change the title of this article but not entirely sure of how to properly do that (without potentially breaking a link in freshdesk somewhere)

### CODE REVIEW

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 1]
```diff
@@ -2,94 +2,60 @@
...
-	Alexis     |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Ariel      |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aubrey     |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Baryshnikov|42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bahia      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Angstrom   |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Asoka      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alcatraz   |41 (0.40%) ++++++++++++++++++++++++++++++++++++++++++++++++++++
+For some OSG Connect users, it may be necessary or advatageous to install your own software. This guide will 
+describe steps and important considerations for transferring your installed software via the HTCondor submit file.
```
Comment (ChristinaLK) [inline start/software/software-transfer-via-htcondor-or-http.md]:
```suggestion
describe steps and important considerations for transferring your installed software to a running job via the HTCondor submit file.  

```

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 2]
```diff
@@ -2,94 +2,60 @@
...
+## Important Considerations
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <100MB staged in your `/home` directory or 
+files >100MB but <1GB staged in your `/public` directory should be transferred via HTCondor using the steps described below. Please see [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985) for more information regarding our policies for staging files (data, software, etc.) in OSG Connect.  
+
+Keep in mind that HTCondor will always transfer the script or binary that is specified as the `executable` in your submit 
+file. Thus, it is not necessary to follow the steps below if your software is the designated executable in your submit file.  
+
+The steps and considerations described below also apply to transferring data, and other files, as described in the [Transfering Data With HTCondor](https://support.opensciencegrid.org/support/solutions/articles/5000639787) guide.
```
Comment (ChristinaLK) [inline start/software/software-transfer-via-htcondor-or-http.md]:
```suggestion
The steps and considerations described below also apply to transferring data, and other files, as described in the [Transferring Data With HTCondor](https://support.opensciencegrid.org/support/solutions/articles/5000639787) guide.

```

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 3]
```diff
@@ -2,94 +2,60 @@
...
-	Angstrom   |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Asoka      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alcatraz   |41 (0.40%) ++++++++++++++++++++++++++++++++++++++++++++++++++++
+For some OSG Connect users, it may be necessary or advatageous to install your own software. This guide will 
+describe steps and important considerations for transferring your installed software via the HTCondor submit file.  
 
+## Important Considerations
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <100MB staged in your `/home` directory or
```
Comment (ChristinaLK) [inline start/software/software-transfer-via-htcondor-or-http.md]: I would simplify to "only software < 1GB should be..."
The later part of the guide talks about `/home` and `/public`.

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 4]
```diff
@@ -2,94 +2,60 @@
...
+> ```
+> transfer_input_files = /home/username/path/to/my_software.tar.gz
+> ```
+> Where `username` refers to your OSG Connect username.
+
+In addition to any software needed for your jobs, be sure to also include other required input files, for example:
+
+	transfer_input_files = path/to/my_software.tar.gz, path/to/my_input.csv
+
+## Transferring Software from `/public`
```
Comment (ChristinaLK) [inline start/software/software-transfer-via-htcondor-or-http.md]: from `/public` or the internet?

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 5]
```diff
@@ -2,94 +2,60 @@
...
+
+Software executables and tar archives that are >100MB should be staged in your `/public` directory. Only software  
+located in `/public` that is <1GB should be transferred using `transfer_input_files` in your submit file. If you have software that is >1GB please plan to use [StashCache](https://support.opensciencegrid.org/support/solutions/articles/12000002775). 
+
+Transferring software from `/public` using the `transfer_input_files` statement in your HTCondor submit file will occur via 
+an HTTP connection, for example:
+
+	transfer_input_files = http://stash.osgconnect.net/public/username/path/my_software.tar.gz
+
+Where `username` refers to your OGS Connect username.
```
Comment (ChristinaLK) [inline start/software/software-transfer-via-htcondor-or-http.md]:
```suggestion
Where `username` refers to your OSG Connect username. 

```

File: start/software/software-transfer-via-htcondor-or-http.md (hunk 4 above)
Comment (jmvera255) [reply start/software/software-transfer-via-htcondor-or-http.md]: yeah, I wasn't sure what to use for the header b/c of the blast precompiled  binary example, maybe make another section with header for "... From the Internet" and then can also include the jar transfer example?

File: start/software/software-transfer-via-htcondor-or-http.md (hunk 5 above)
Comment (jmvera255) [reply start/software/software-transfer-via-htcondor-or-http.md]: nice catch!

File: start/software/software-transfer-via-htcondor-or-http.md (hunk 4 above)
Comment (ChristinaLK) [reply start/software/software-transfer-via-htcondor-or-http.md]: On further thought, I would just stick w/ `/public` and either just mention that it also works for other links, or don't cover that at all.

**Comment by jmvera255** [comment]
I have fixed the typos that @ChristinaLK identified and made some additional revisions

**Comment by ChristinaLK** [comment]
Looks good! Merging.

File: start/software/software-transfer-via-htcondor-or-http.md [hunk 6]
```diff
@@ -2,94 +2,63 @@
...
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <1GB should be transferred via HTCondor using the steps described below. Please see [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985) for more information regarding our policies for staging files (data, software, etc.) in OSG Connect.  
+
+Keep in mind that HTCondor will always transfer the script or binary that is specified as the `executable` in your submit 
+file. Thus, it is not necessary to follow the steps below if your software is the designated executable in your submit file.  
+
+The steps and considerations described below also apply to transferring data, and other files, as described in the [Transferring Data With HTCondor](https://support.opensciencegrid.org/support/solutions/articles/5000639787) guide.
+
+## Transferring Software From `/home`
```
Comment (lmichael107) [inline start/software/software-transfer-via-htcondor-or-http.md:22]: Use a title that states an objective, not the solution, e.g. "Transfer Data or Software Smaller than XXMB":

**Pull request merged**